src/Pact/Analyze/Prisms.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PatternSynonyms   #-}
+{-# LANGUAGE Rank2Types        #-}
+{-# LANGUAGE ViewPatterns      #-}
+
+module Pact.Analyze.Prisms where
```

Comment:
seems like this could be either `Pact.Analyze.Prisms` or `Pact.Analyze.Feature.Prisms`